experimenter/experimenter/settings.py

```diff
@@ -530,6 +530,14 @@
     BASE_DIR / "outcomes" / "metric-hub-main" / "jetstream" / "outcomes"
 )
 
+JETSTREAM_CONFIG_SEGMENTS_PATH = (
```

Comment:
So the `JETSTREAM_CONFIG` part of the other name is actually a relic of pre-metric-hub config repo, which was called jetstream-config. Maybe it's time to update these settings to say `METRIC_HUB_` instead of `JETSTREAM_CONFIG_`? This would make all three of these related settings consistent, and update them for the new repo.

Something like:
- `METRIC_HUB_OUTCOMES_PATH`
- `METRIC_HUB_SEGMENTS_PATH_DEFAULT`
- `METRIC_HUB_SEGMENTS_PATH_JETSTREAM`